examples/data-fetch/package.json

```diff
@@ -7,7 +7,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "isomorphic-fetch": "^2.2.1",
+    "cross-fetch": "^0.0.8",
```

Comment:
Why change out a well tested lib to a lib that is not even 1.0.0 and does exactly the same thing?